src/Microsoft.OpenApi.Readers/OpenApiReaderSettingsExtensions.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.OpenApi.Interfaces;
+using Microsoft.OpenApi.Models;
+using Microsoft.OpenApi.Readers;
+
+namespace Microsoft.OpenApi.Reader;
```

Comment:
Out of curiosity, should this class exist in a different namespace as the YamlReader?